addons/mail/static/src/core/common/message.xml

```diff
@@ -180,7 +180,7 @@
         <t t-if="isMobileOS and isReverse" t-call="mail.Message.emptyQuickAction"/>
         <div class="d-flex rounded-1 overflow-hidden gap-1" t-att-class="{ 'flex-row-reverse': isReverse }">
             <ActionList actions="actions" inline="true" fw="false"/>
-            <t t-foreach="Array.from({ length: quickActionCount - quickActions.length - 1 })" t-as="emptyQuickAction" t-key="emptyQuickAction_index">
```

Comment:
`quickActions` is a `slice(quickActionCount)` so theoretically should not require `Math.max`, but considering that `message.xml` is a slotted component with `ActionSwiper`, I wonder if you observe a crash because of this?